cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/CaReconcilerTest.java

```diff
@@ -327,7 +327,7 @@ public void testReconcileCasWhenCustomCertsAreMissingThrows(Vertx vertx, VertxTe
         reconcileCa(vertx, certificateAuthority, certificateAuthority)
             .onComplete(context.failing(e -> context.verify(() -> {
                 assertThat(e, instanceOf(InvalidResourceException.class));
-                assertThat(e.getMessage(), is("Cluster CA should not be generated, but the secrets were not found."));
+                assertThat(e.getMessage(), is("cluster-ca should not be generated, but the secrets were not found."));
```

Comment:
In the docs we call it `Cluster CA` So why not keep it in the exception message?